srcpkgs/memtest86+/template

```diff
@@ -1,8 +1,8 @@
 # Template file for 'memtest86+'
 pkgname=memtest86+
 version=5.01
-revision=5
-archs="i686 x86_64"
+revision=6
+archs="i686* x86_64*"
```

Comment:
I think instead of cross build you meant musl build?

Reply:
Yes, well, actually yes...